### DISCUSSION TITLE
At this rate we are going to reach FNF v0.9 in no time, so will it go to FNF v1 or FNF v0.10?

### DISCUSSION THREAD

**Discussion by macohi** [FunkinCrew/Funkin#3797] [Q&A] [answered]
At the current pace I honestly expect us to reach FNF v0.9.x very soon.

But my main question is if it will go to FNF v1 or FNF v0.10.

Just a random question I had.

**Comment by AbnormalPoof** [comment, marked as answer]
Hope this answers your question: https://x.com/EliteMasterEric/status/1801052716149432761

**Reply by macohi** [reply]
it does, thanks!

**Comment by PSI3x** [comment]
1.0 would probably be the paid game

**Comment by ChillyBeanBAM** [comment]
I think jumping straight to v0.9.x from v0.6.x is a stretch, but there's a small possibility.

**Reply by macohi** [reply]
Oh,
Yeah I see what you mean
im bad at realistic estimation lols

**Comment by ghost** [comment]
what?
